calculator_test.go

```diff
@@ -2,9 +2,18 @@ package calculator_test
 
 import (
 	"calculator"
+	"math/rand"
 	"testing"
+	"time"
 )
 
+type testCase struct {
+	a, b        float64
+	want        float64
+	name        string
```

Comment:
We might put the name first, to make the cases a bit more readable, mightn't we?